vm/src/bytecode.rs

```diff
@@ -42,6 +42,7 @@ pub enum Instruction {
     Import {
         name: String,
         symbol: Option<String>,
+        source_path: Option<String>,
```

Comment:
Sourcepath should be part of the codeobject, not the import bytecode. In the vm, the current codeobject can be queried for its sourcepath when executing.

Reply:
Btw, sourcepath is also needed for exception tracebacks :+1: